taskcluster/ci/signing/kind.yml

```diff
@@ -24,15 +24,11 @@ only-for-build-types:
 
 job-template:
     description: Sign component
-    worker-type:
-        by-level:
-            '3': signing
-            default: dep-signing
-    worker:
-        signing-type:
-            by-level:
-                '3': release-signing
-                default: dep-signing
+    index:
+        by-build-type:
+            (release|nightly):
```

Comment:
No need to handle snapshots since they're deprecated.

Reply:
Note to self: I've got some local patches that I haven't put up for review to remove all of the snapshots logic. I just wanted to wait for another week before landing. But this cleanup sounds good, might as well do it now since we're here 👍